thirdparty/README.md

```diff
@@ -173,6 +173,22 @@ comments. Apply the patches in the `patches/` folder when syncing on newer upstr
 commits.
 
 
+## glad
+
+- Upstream: https://github.com/Dav1dde/glad
+- Version: 2.0.2 (f237a2bfcec0d9b82b90ec9af4af265c40de7183, 2022)
+- Generator License: MIT, Apache, Khronos
+- Generated files license: PD/WTFPL/CC0, Apache
+
+Files extracted from upstream source:
+- `LICENSE`
```

Comment:
Since we don't actually include glad's code, I guess we could skip the `LICENSE` file altogether, as there's no MIT code in this repo, nor EGL specification (so far?). We could maybe trim it to just the ASL section for the Khronos GL spec.